### PR TITLE
For #12531: Align Settings -> Private browsing to 72dp keyline

### PR DESCRIPTION
**Pull request by hakkikaancaliskan**
cc @brampitoyo

Screenshot_1594712979

### Pull Request checklist

- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [x] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/res/xml/private_browsing_preferences.xml [hunk 1]
```diff
@@ -2,20 +2,16 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto">
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
     <androidx.preference.Preference
```
Comment (NotWoods) [inline app/src/main/res/xml/private_browsing_preferences.xml:6]: I noticed that you remove the namespace in some of the other PRs, can it be removed here? (not needed)

Comment (hakkikaancaliskan) [reply app/src/main/res/xml/private_browsing_preferences.xml:6]: Android Studio warns me on not used ones, then I'm removing them. `xmlns:android` is used in there.

Comment (NotWoods) [reply app/src/main/res/xml/private_browsing_preferences.xml:6]: Sorry, I meant the usage here:
```suggestion
    <Preference
```
Can it get removed?

Comment (hakkikaancaliskan) [reply app/src/main/res/xml/private_browsing_preferences.xml:6]: Ahhh yes we can :)

**Comment by brampitoyo** [comment]
@hakkikaancaliskan Looks good to me, as well!

**Comment by hakkikaancaliskan** [comment]
@NotWoods could you please label this as needs landing?

**Comment by person808** [comment]
Closing for #12654

**Pull request closed without merging**